docs/docs/general/02-Components/AskUI-Development-Environment.md

```diff
@@ -185,3 +186,74 @@ Note: Adjust the parameters as needed for your specific configuration.
 :::
 
 See also the dedicated [AskUI Runner docs](AskUI-Runner.md) for more information.
+
+
+## ADE + VSCode IDE
+
+The ADE is build to work seamlessly with (VSCode)[https://code.visualstudio.com/] together. Therefore we create (VScode Workspace Settings)[https://code.visualstudio.com/docs/getstarted/settings#_settingsjson]
+
+**Note: Only compatible with windows**
+
+1. Create a `<project>/.vscode` folder with `mkdir .vscode`
+
+### Configure the askui-shell
+
+2. Create the `<project>/.vscode/settings.json` with following content:
+```json
+{
+    "livePreview.customExternalBrowser": "Default",
+    "terminal.integrated.profiles.windows":{   
+        "askui-shell": {    
+            "path": ["C:\\Program Files\\AskUI GmbH\\AskUI Suite\\Tools\\askui-shell.cmd"],
+            "args": [
+               "activate"
+            ],
+            "icon": "terminal-bash",
+        },        
+        "Git Bash": null,        
+        "Command Prompt": null,        
+        "PowerShell": null,        
+        "JavaScript Debug Terminal": null
+    },
+    "terminal.integrated.defaultProfile.windows": "askui-shell"
+}
+```
+- `terminal.integrated.profiles.windows`: Configures the `askui-shell` and deactivate other shells
+- `terminal.integrated.defaultProfile.windows`: Set the `askui-shell` as default
+
+Then the `askui-shell` is configured for as the default terminal like this:
+![askui logo](./docs/static/img/vs-code/askui_shell.png)
+
+
+### Configure Jest Runner, ESLint & Live View
+
+Add the recommended extenstions for VSCode by adding `<project>/.vscode/extensions.json` with following content:
+
+```json
+{
+    "recommendations": ["ms-vscode.live-server", "firsttris.vscode-jest-runner", "dbaeumer.vscode-eslint"]
+}
+```
+- `ms-vscode.live-server`: HTML [Live Viewer](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server) for annotations.
```

Comment:
I would try to explain the extensions or, more specifically, what they enable the user to do, in more detail here as this is the first time they are mentioned and this would be what a user wonders over initially.

```suggestion
- `ms-vscode.live-server`: HTML [Live Viewer](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server) for viewing annotations inside VSCode instead of, e.g., having to jump out of VSCode and opening it up in a web browser.
```